model/schema/core.yaml

```diff
@@ -355,71 +356,74 @@ classes:
       - prefix_order
 
   Chromosome:
-    is_a: AuditedObject
-    description: The ID of the landmark used to establish the coordinate system for the current feature.
+    is_a: GenomicEntity
+    description: >-
+      A structure composed of a single DNA molecule and associated proteins
+      that carries here heriditary information from cell to cell. In the context
+      of this model, it is used as a landmark for cytogenetic and genetic
+      mapping of genes, alleles and chromosomal aberrations; this entity ties
+      together related chromosome sequences from different genome assemblies.
+    notes: >-
+      I'm trying to distinguish the idealized biological entity from a species'
+      genome from a chromosome sequence assembly that is specific to a given
+      genome assembly version.
+    exact_mappings:
+      - SO:0000340
+      - GO:0005694
+
+  ChromosomeAssembly:
```

Comment:
I wonder if the name should be something more generic, e.g. AssemblyComponent.  I'm thinking that in the various places that we are likely to use this we are probably going to really mean scaffolds or contigs in addition to chromosomes.

Reply:
I agree and I'll make the change. Going with this new definition: A` scaffold component of a genome assembly, including chromosomal pseudo-molecule sequences, unplaced/unlocalized scaffolds and contigs, and organelle genomes.` On NCBI Assembly pages, the generic term `scaffold` includes the chromosomal assemblies, so I used that in the definition, but I can change that if that seems wrong.